react/src/pages/VFolderListPage.tsx

```diff
@@ -40,20 +40,20 @@ const tabParam = withDefault(StringParam, 'general');
 
 const VFolderListPage: React.FC<VFolderListPageProps> = (props) => {
   const { t } = useTranslation();
+  const { token } = theme.useToken();
   const [curTabKey, setCurTabKey] = useQueryParam('tab', tabParam, {
     updateType: 'replace',
   });
-  const baiClient = useSuspendedBackendaiClient();
   const [fetchKey, updateFetchKey] = useUpdatableState('first');
-  const dataViewRef = useRef(null);
   const [isOpenCreateModal, { toggle: openCreateModal }] = useToggle(false);
   const [inviteFolderId, setInviteFolderId] = useState<string | null>(null);
   const [
     isVisibleImportFromHuggingFaceModal,
     { toggle: toggleImportFromHuggingFaceModal },
   ] = useToggle(false);
 
-  const { token } = theme.useToken();
+  const dataViewRef = useRef(null);
+  const baiClient = useSuspendedBackendaiClient();
   const enableImportFromHuggingFace =
```

Comment:
Is there any reason to move these code?

Reply:
The variable declaration has been modified to make it easier to see. The token is included on line 42.